products/memgraph/upcoming/tutorials/replication.md

```diff
@@ -0,0 +1,136 @@
+## Replication
+
+This article is a part of a series intended to showcase Memgraph's features and
+on realistic scenarios, and bring the user up to speed on developing with
+Memgraph.
+
+We highly recommend checking out the other articles from this series which
+are listed in our [tutorial overview section](tutorials.md).
+
+### Introduction
+
+This example demonstrates how to create a simple cluster of nodes running
+Memgraph instances, and set up replication. We'll demonstrate the use and
+behavior of the available synchronization modes.
+
+### Cluster topology
+
+The cluster consists of four nodes, a single main and three replicas:
+
+* the main node, containing the original data to be replicated to other nodes,
+* a node that will be replicated to using the sync mode,
+* a node that will be replicated to using the async mode,
+* a node that will be replicated to using the sync-with-timeout mode .
+
+We'll be hitting the main node with create queries, creating nodes and edges,
+while running BFS on the replicas, and observing the effects of different sync
+modes used.
+
+### Configuring the cluster
+
+We'll use Docker to set up and run the cluster on your local machine, so make
+sure you have it installed and ready, and grab your Memgraph docker image or
+binary. Look [here](https://docs.memgraph.com/memgraph/getting-started/installation/docker-installation) for instructions.
+In addition, make sure you have either [mg_client](https://github.com/memgraph/mgclient/tree/release/1.0) or [mgconsole](https://github.com/memgraph/mgconsole) installed.
```

Comment:
Replace `mg_client` with `mgclient` + the URL should be https://github.com/memgraph/mgclient (latest version should be used if possible). `mgclient` is the actual client, `mg_client` is a tool still shipped with Memgraph, but it will be removed in the future.

Even better would be to leave just `mgconsole` here for the sake of simplicity because only that is really needed to showcase how to set up the replication cluster.